sdk/core/Azure.Core.Expressions.DataFactory/api/Azure.Core.Expressions.DataFactory.netstandard2.0.cs

```diff
@@ -56,7 +56,7 @@ public DataFactoryLinkedServiceReference(Azure.Core.Expressions.DataFactory.Data
         public static Azure.Core.Expressions.DataFactory.DataFactoryLinkedServiceReferenceKind LinkedServiceReference { get { throw null; } }
         public bool Equals(Azure.Core.Expressions.DataFactory.DataFactoryLinkedServiceReferenceKind other) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override bool Equals(object obj) { throw null; }
+        public override bool Equals(object? obj) { throw null; }
```

Comment:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.Expressions.DataFactory/src/Azure.Core.Expressions.DataFactory.csproj